spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/metrics/SparkBigQueryReadSessionMetrics.java

```diff
@@ -30,52 +31,58 @@
 
 public class SparkBigQueryReadSessionMetrics extends SparkListener
     implements Serializable, ReadSessionMetrics {
-  private static final String bytesRead = "bqBytesRead";
```

Comment:
Please keep the constants, and give them java style names (e.g. BYTES_READ)

Reply:
Done